core/tests/test_nothing.py

```diff
@@ -0,0 +1,6 @@
+from unittest import TestCase
+
+
+class NothingTest(TestCase):
```

Comment:
I'm not fully certain if at the end I'll still have code to test in core. But everything else to run tests is already there, and I found it weird to have the dependencies on pytest, the Make action, etc. and not the job. Error-prone also because someone adding tests will not think of the CI job if everything else is there... But I could remove the extra things instead, considering the probabilities that there will be tests are low.

Reply:
Ok, let's keep it then. Especially since I think we should have unit tests for the `UnsupportedIntentPolicy`